config/PropertyNames.xml

```diff
@@ -507,30 +508,30 @@ generated from the section label.
         <property name="Node.Trace.Server" />
         <property name="Node.UserAccounts" />
         <property name="Node.UserAccountMapper" class="proxy" />
-        <property name="Node.WaitTime" />
+        <property name="Node.WaitTime" default="60" />
         <property name="Registry.AdminCryptPasswords" />
         <property name="Registry.AdminPermissionsVerifier" class="proxy" />
-        <property name="Registry.AdminSessionFilters" />
+        <property name="Registry.AdminSessionFilters" default="0" />
         <property name="Registry.AdminSessionManager" class="objectadapter" />
         <property name="Registry.AdminSSLPermissionsVerifier" class="proxy" />
         <property name="Registry.Client" class="objectadapter" />
         <property name="Registry.CryptPasswords" />
         <property name="Registry.DefaultTemplates" />
         <property name="Registry.Discovery" class="objectadapter" />
-        <property name="Registry.Discovery.Enabled" />
+        <property name="Registry.Discovery.Enabled" default="1" />
         <property name="Registry.Discovery.Address" />
-        <property name="Registry.Discovery.Port" />
+        <property name="Registry.Discovery.Port" default="4061" />
         <property name="Registry.Discovery.Interface" />
         <property name="Registry.DynamicRegistration" />
         <property name="Registry.Internal" class="objectadapter" />
         <property name="Registry.LMDB.MapSize" />
         <property name="Registry.LMDB.Path" />
         <property name="Registry.PermissionsVerifier" class="proxy" />
-        <property name="Registry.ReplicaName" />
+        <property name="Registry.ReplicaName" default="Master" />
         <property name="Registry.RequireNodeCertCN" />
         <property name="Registry.RequireReplicaCertCN" />
         <property name="Registry.Server" class="objectadapter" />
-        <property name="Registry.SessionFilters" />
+        <property name="Registry.SessionFilters" default="0" />
```

Comment:
For all the Trace below, the default is not 0?

Reply:
Probably, I haven't gotten to properties that we don't read plainly. 
For instance grepping for `Node.Trace.Activator` doesn't show anything.

```
❯ rg -F Node.Trace.Activator
config/PropertyNames.xml
504:        <property name="Node.Trace.Activator" />

scripts/IceGridUtil.py
131:            "IceGrid.Node.Trace.Activator": 0,

java/src/Ice/src/main/java/com/zeroc/IceInternal/PropertyNames.java
810:    new Property("IceGrid.Node.Trace.Activator", false, "", false, null),

csharp/src/Ice/Internal/PropertyNames.cs
784:         new(@"IceGrid.Node.Trace.Activator", false, "", false, null),

cpp/config/icegridnode.cfg
58:IceGrid.Node.Trace.Activator=3

cpp/src/Ice/PropertyNames.cpp
802:    IceInternal::Property("IceGrid.Node.Trace.Activator", false, "", false, nullptr),
```

Reply:
https://github.com/zeroc-ice/ice/blob/c4a96d3ea88cccd593341cc86b46732c54a468a6/cpp/src/IceGrid/TraceLevels.cpp#L47